client/src/pages/dashboards/DiagramNode.js

```diff
@@ -0,0 +1,309 @@
+import { css, keyframes } from "@emotion/core"
+import styled from "@emotion/styled"
+import {
+  AbstractModelFactory,
+  AbstractReactFactory
+} from "@projectstorm/react-canvas-core"
+import {
+  NodeModel,
+  PortModel,
+  PortModelAlignment,
+  PortWidget
+} from "@projectstorm/react-diagrams-core"
+import {
+  DefaultLinkFactory,
+  DefaultLinkModel,
+  DefaultLinkWidget
+} from "@projectstorm/react-diagrams-defaults"
+import AggregationWidgetContainer, {
+  AGGERGATION_WIDGET_TYPE,
+  getAggregationWidget
+} from "components/aggregations/AggregationWidgetContainer"
+import AvatarDisplayComponent from "components/AvatarDisplayComponent"
+import { SPECIAL_WIDGET_TYPES } from "components/CustomFields"
+import LinkTo from "components/LinkTo"
+import { CUSTOM_FIELD_TYPE } from "components/Model"
+import { GRAPHQL_NOTES_FIELDS } from "components/RelatedObjectNotes"
+import _uniqueId from "lodash/uniqueId"
+import * as Models from "models"
+import moment from "moment"
+import { PERIOD_FACTORIES, RECURRENCE_TYPE } from "periodUtils"
+import PropTypes from "prop-types"
+import * as React from "react"
+
+const ENTITY_GQL_FIELDS = {
+  Report: "uuid, intent",
+  Person: `uuid, name, role, avatar(size: 64), ${GRAPHQL_NOTES_FIELDS}`,
+  Organization: "uuid, shortName",
+  Position: "uuid, name",
+  Location: "uuid, name",
+  Task: `uuid, shortName, longName, ${GRAPHQL_NOTES_FIELDS}`
```

Comment:
Commas are not needed to separate GQL fields; just whitespace is preferred.
(Yes, I am aware there are other GQL requests that [still] have commas, but we should at least not add new ones.)

Also, they appear to be largely the same (apart from the notes fields) as `ENTITY_GQL_FIELDS` in `LinkAnetEntity.js`, so perhaps the common part should be extracted.